sdk/data/aztables/CHANGELOG.md

```diff
@@ -5,6 +5,13 @@
 ### Features Added
 
 ### Breaking Changes
+* Prefixed all `TransactionType` constants with `TransactionType`.
+* Prefixed all `EntityUpdateMode` constants with `EntityUpdateMode`.
+* Changed the `SharedKeyCredential.ComputeHMACSHA256` method to a private method.
+* Changed the `ListTablesPager` and `ListEntitiesPager` to structs.
+* Renamed the `ResponseProperties` type to `TableProperties`.
+* Returns `Client` and `ServiceClient` by value instead of by reference.
```

Comment:
Why? Client constructors should return pointers.

Reply:
Fixed

Reply:
Mis-remembered the guidelines